configs/datamodule/herwig.yaml

```diff
@@ -5,4 +5,4 @@ pin_memory: False
 core_dataset:
   _target_: hadml.datamodules.components.herwig.Herwig
   data_dir: "${paths.data_dir}Herwig"
-  train_val_test_split: [0.96, 0.02, 0.02]
+  train_val_test_split: [80, 10, 10]
```

Comment:
Can we change it to `[800, 100, 100]`? It's not very important, but this way it would be clear those aren't percentage points